DatWeatherDoe/UI/Menu Bar/StatusItemManager.swift

```diff
@@ -20,6 +20,18 @@ final class StatusItemManager {
     init(menu: NSMenu, configureSelector: Selector) {
         statusItem.menu = menu
         statusItem.button?.action = configureSelector
+
+        // Category icons in dropdown menu
+        locationMenuItem?.image = WeatherConditionImageMapper().map(.location)
+        temperatureForecastMenuItem?.image = WeatherConditionImageMapper().map(.thermometer)
+        sunRiseSetMenuItem?.image = WeatherConditionImageMapper().map(.sunny)
+        windMenuItem?.image = WeatherConditionImageMapper().map(.windy)
+        // By making the images "templates", macOS knows to adjust the image color based on
+        // the desktop settings (light, dark), so they remain visible either way.
+        locationMenuItem?.image?.isTemplate = true
```

Comment:
These lines should move next to the menuItem that's being operated on for better readability. So something like this:

locationMenuItem?.image = ...
locationMenuItem?.image?.isTemplate = true
...
temperatureForecastMenuItem?.image = 
temperatureForecastMenuItem?.image?.isTemplate ...

Reply:
Sure.